src/cleanlab_tlm/utils/chat.py

```diff
@@ -214,6 +265,7 @@ def _find_index_after_first_system_block(messages: list[dict[str, Any]]) -> int:
 def _form_prompt_responses_api(
     messages: list[dict[str, Any]] | str,
     tools: Optional[list[dict[str, Any]]] = None,
+    response: Optional[Response] = None,
```

Comment:
this needs to be documented in docstring, especially why somebody might sometimes choose to add this in a method called form_prompt() - which seems curious

Reply:
also does this change mean that `form_prompt_string()` no longer properly works for Responses API messages?  Would want to call that out too somehow.